build.gradle

```diff
@@ -110,5 +143,14 @@ jacocoTestReport {
 
 check.dependsOn jacocoTestReport
 
+integrationTest {
+    useJUnitPlatform()
+}
+
+wrapper {
```

Comment:
Last question: what is the rationale for this ?

Reply:
Removed. I have copied it from other gradle configuration, now I see it is not necessary.